content/sigs/advocacy-and-outreach/outreach-programs/index.adoc

```diff
@@ -0,0 +1,85 @@
+---
+layout: project
+title: "Outreach programs"
+section: sigs
+sigId: "advocacy-and-outreach"
+tags:
+  - advocacy-and-outreach
+  - advocacy-and-outreach-sig
+  - outreach-programs
+  - community
+  - outreach
+  - programs
+---
+
+Jenkins participates in the following community programs:
+
+# Google Summer of Code
+
+image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=right]
+
+link:https://developers.google.com/open-source/gsoc/[Google Summer of Code]
+(GSoC) is an annual, international, program which encourages
+college-aged students to participate with open source projects during the summer
+break between classes.
+
+Students accepted into the program receive a stipend,
+paid by Google, to work well-defined projects to improve or enhance the Jenkins project.
+In exchange, numerous Jenkins community members volunteer as "mentors"
+for students to help integrate them into the open source community and succeed
+in completing their summer projects.
+
+To contact us regarding this program,
+send us a message on the
+link:https://groups.google.com/forum/#!forum/jenkinsci-gsoc-all-public[GSoC mailing list]
+or contact us on the
+link:https://gitter.im/jenkinsci/gsoc-sig:[GSoC Gitter channel].
+
+// The GSoC logo is a bit tall, so add some empty lines
+{empty} +
+
+* * *
+
+# Google Season of Docs
+
```

Comment:
There might be some confusion, because we gave never been accepted to GSoD. Not sure we "participated"

Reply:
Agreed.  I'm in favor of removing the reference to Google Season of Docs until the first time we're accepted to the program.